### DISCUSSION TITLE
Does nextcord support slash command?

### DISCUSSION THREAD

**Discussion by MisileLab** [nextcord/nextcord#90] [Q&A] [answered]
Does nextcord support slash command?

**Comment by TheMultii** [comment, marked as answer]
Not yet

**Reply by TheMultii** [reply]
Well, unless you mean support for external libraries. There is a chance that they will work 🤔

**Reply by MisileLab** [reply]
> Well, unless you mean support for external libraries. There is a chance that they will work 🤔

finally I made external library base, when I send to it to github, I'll give you github link
